test/integration/test_workflow_invocation.py

```diff
@@ -41,7 +41,7 @@ def test_run_workflow_with_missing_tool(self):
             )
             self._assert_status_code_is(invocation_response, 400)
             assert (
-                invocation_response.json().get("err_msg")
+                invocation_response.json().get("detail")
```

Comment:
Will this change break existing API consumers expecting the information to be in `err_msg`?